docs/tutorials/04_excited_states_solvers.ipynb

```diff
@@ -115,21 +119,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 3,
```

Comment:
I cannot mark this on the text above - but this could do with improving - the `explaining the algorithms how to find the ground state` bit mainly

>To use the qEOM as a solver in Qiskit, we have to define a ground state calculation first, explaining the algorithms how to find the ground state.

Reply:
done